src/lib/containers/IconSvg.tsx

```diff
@@ -151,15 +188,17 @@ export type SVGStyleProps = {
   color?: string
   verticalAlign?: string
   fill?: string
+  width?: string
 }
 
 const getIcon = (options: IconSvgOptions) => {
-  const { icon, color } = options
+  const { icon, color, size } = options
```

Comment:
Setting size didn't seem to do anything to the material icons.  Now it sets the width of the svg (so it has a direct impact on the size since the aspect is maintained)